packages/gluestick/src/config/__tests__/compileWebpackConfig.test.js

```diff
@@ -1,8 +1,13 @@
-/* @flow */
```

Comment:
Any benefits of adding it if we don't have any typings in this file. We would only need to ignore `process.cwd` bindings.